integrations-and-supported-tools/prophet/notebooks/Neptune_prophet.ipynb

```diff
@@ -0,0 +1,343 @@
+   "id": "8f521acb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run = neptune.init(\n",
+    "    project='common/fbprophet-integration', \n",
+    "    api_token=\"ANONYMOUS\",\n",
+    "    tags=[\"fbprophet\", \"artifacts\", \"notebook\"], # optional  \n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9cf32479",
+   "metadata": {},
+   "source": [
+    "Click on the link above to open this run in Neptune. For now it is empty but keep the tab with run open to see what happens next.\n",
+    "\n",
+    "You tell Neptune: \n",
+    "\n",
+    "* **who you are**: your Neptune API token `api_token` \n",
+    "* **where you want to send your data**: your Neptune `project`.\n",
+    "\n",
+    "At this point you have new run in Neptune. For now on you will use `run` to log metadata to it.\n",
+    "\n",
+    "---\n",
+    "\n",
+    "**Note**\n",
+    "\n",
+    "\n",
+    "Instead of logging data to the public project `'common/sklearn-integration'` as an anonymous user 'neptuner' you can log it to your own project.\n",
+    "\n",
+    "To do that:\n",
+    "\n",
+    "1. Get your [Neptune API token](https://docs.neptune.ai/getting-started/installation#get-api-token)\n",
+    "2. Pass the token to ``api_token`` argument of ``neptune.init()`` method: ``api_token=YOUR_API_TOKEN``\n",
+    "3. Pass your project to the ``project`` argument of ``neptune.init()``.\n",
+    "\n",
+    "For example:\n",
+    "\n",
+    "```python\n",
+    "neptune.init(project=\"YOUR_WORKSPACE/YOUR_PROJECT\",\n",
+    "             api_token=\"YOUR_API_TOKEN\")\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "702d8c5b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "21659dce",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = pd.read_csv(\"https://raw.githubusercontent.com/facebook/prophet/master/examples/example_wp_log_R.csv\")\n",
+    "df[\"cap\"] = 8.5"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8180d30a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from prophet import Prophet"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dd382b5f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = Prophet(growth='logistic')\n",
+    "model.fit(df)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9f8579f7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "forecast = model.predict(df)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "307f8733",
+   "metadata": {},
+   "source": [
+    "### Log summary metadata"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "98c5a2cb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import neptune_prophet.impl as npt_utils\n",
```

Comment:
How do we want to do this? It's inconsistent with the examples in the docstrings.

cc: @twolodzko